src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -98,7 +96,7 @@ export const Introduction: React.FC<Props> = ({
         </Hero>
         <HeroContentWrapper>
           <ContentContainer>
-            <OfferDetails />
+            <HeroOfferDetails {...{ offerData }} />
```

Comment:
offerData={offerData}?

Reply:
You're right, that's how we write it! 🧠  Just an old habit of mine slipping through I guess (or however one puts it)